pixart.js

```diff
@@ -1 +1,36 @@
+//add 2 more brushes
+$(".brush").attr("class", "brush 1");
+$(".controls").append("<div class='brush 2'></div>");
+$(".controls").append("<div class='brush 3'></div>");
```

Comment:
Not sure if you did this intentionally, but remember that `class="brush 2"` is interpreted as two classes: `brush` and `2`.